README.md

```diff
@@ -0,0 +1,29 @@
+# Lost
+
+Are you truly lost? Will you ever return home?
+A researcher on a space expedition is stranded on an alien world with no way to get home.  
+
+
+ ## How To Play
+
+Download the Lost module and it's dependacies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.
+
+## Current Features
+
+### Special Structure Generation
+
+Providing the story of the Lost World.
```

Comment:
Not sure what you mean here?

Reply:
I was just guessing, I haven't tested Lost enough to know about the set pieces that spawn.

Reply:
Replacing section with better details